content/en/continuous_integration/tests/_index.md

```diff
@@ -89,9 +89,43 @@ You can also interactively plot graphs and top lists using the [Analytics][6] ta
 
 Spans generated by third party services that are instrumented with APM and that are involved in integration tests appear in [APM][7]. You can filter spans that were generated as part of an integration test using the `Origin Service` facet and selecting the test service name used by the integration test.
 
+### Test suite level visibility
+
+In addition to tests, you have visibility over the whole testing phase of your project. On the [Test Runs][4] page, you can filter by test level: session, module, suite and test. Each test level represents a different level of aggregation of tests.
+
+{{< img src="ci/ci-test-suite-visibility.png" alt="Test Suite Visibility" style="width:100%;">}}
+
+#### Sessions
+Test sessions are the highest level of aggregation. They correspond one to one to a test command:
```

Comment:
while this is not true for Swift, I'm not sure whether it makes sense to talk about that here.